doc/source/whatsnew/v0.17.0.txt

```diff
@@ -69,6 +67,13 @@ Other enhancements
 
 - Enable `read_hdf` to be used without specifying a key when the HDF file contains a single dataset (:issue:`10443`)
 
+- :meth:`~pandas.io.read_sas` provides support for reading SAS XPORT format files:
+
```

Comment:
add this in the highlites section (at the top) as well (you can put a link to the actualy docs there as well)